### ISSUE TITLE
`requirements.txt` missing from PyPI source code

### ISSUE THREAD

**Issue by xylar** [ExaWorks/psij-python#466]
For the last 3 releases (0.9.4, 0.9.5 and 0.9.6), the `requirements.txt` file has not been packaged into the tarball on PyPI.  This means that `psij-python` cannot be installed from source using these tarballs.

See:
https://github.com/conda-forge/psij-python-feedstock/pull/6

**Comment by xylar** [comment]
To reproduce the issue:
```
$ wget https://files.pythonhosted.org/packages/5a/4f/89e063358d2ffd3afde74d85824294dfcc1900255e4571df52b39235177d/psij-python-0.9.6.tar.gz
$ tar xvf psij-python-0.9.6.tar.gz
$ cd psij-python-0.9.6
$ pip install .

Processing /home/xylar/Desktop/pypi/psij-python-0.9.6
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [7 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/home/xylar/Desktop/pypi/psij-python-0.9.6/setup.py", line 8, in <module>
          with open('requirements.txt') as f:
               ^^^^^^^^^^^^^^^^^^^^^^^^
      FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

**Comment by hategan** [comment]
Reopening until release is made and verified

**Comment by hategan** [comment]
0.9.7 is now released and `pip install psij-python-0.9.7.tar.gz` succeeds (where the tar.gz is downloaded from pypi)

**Comment by xylar** [comment]
Thanks so much @hategan!
